examples/scaling-criteo/03-Training-with-Merlin-Models.ipynb

```diff
@@ -0,0 +1,417 @@
+{
```

Comment:
<p>Are you using 4 or 5 days of criteo dataset for training? Not sure you copy this configuration again. I'd simply say `We archived an AUC score of 0.7247 from training, with .. of parquet files. Now let's see how we can easily perform evaluation on our trained model".</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/Merlin/pull/380/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Reply:
The information is removed as we add it to the CI tests later